app/javascript/miq-redux/reducer.ts

```diff
@@ -0,0 +1,25 @@
+import {createStore, Store, Reducer, Action} from 'redux';
+
+import { IMiqAction, AppReducer, IMiqReducerHash, AppState} from '../packs/miq-redux';
+
+const reducers: Set<AppReducer> = new Set();
+
+export const rootReducer: AppReducer = (state = {}, action: IMiqAction) => {
+    let newState = state;
+
+    reducers.forEach((reducer) => {
+        newState = reducer(newState, action)
```

Comment:
Well, it depends on how the API would look like. Right now, there's a simple `addReducer` function. Can you share some code snippet of how you'd imagine the reducer composition in MiQ Redux infra?